fixbackend/dispatcher/dispatcher_service.py

```diff
@@ -96,18 +120,68 @@ async def process_collect_done_message(self, message: Json, context: MessageCont
             case _:
                 log.info(f"Collect messages: will ignore messages of kine {context.kind}")
 
+    def _collect_progress_hash_key(self, workspace_id: WorkspaceId) -> str:
+        return f"dispatching:collect_jobs_in_progress:{workspace_id}"
+
+    async def maybe_send_domain_event(self, tenant_id: WorkspaceId, completed_job_id: str) -> None:
```

Comment:
Consider to rename this method, since it will also mark the job as done etc. not only sending an event eventually.